empiar/__init__.py

```diff
@@ -64,8 +64,4 @@ def defineBinaries(cls, env):
                        commands=empiar_cmd)
 
 
-pyworkflow.em.Domain.registerPlugin(__name__)
-
-
-
-
+pwem.Domain.registerPlugin(__name__)
```

Comment:
Now, the plugins are not registered, entry_points is being used (see how they are defined in the other plugins)